modules/auth.js

```diff
@@ -21,95 +20,89 @@ function authenticate (email, password) {
   })
 }
 
-function signupApi (email, password, password_confirmation, newsletter) { // eslint-disable-line
+function signupApi (email, password, passwordConfirmation, newsletter) {
   return apiRequest.post('account/register', {
     account: {
       email,
       password,
-      password_confirmation,
+      passwordConfirmation,
       newsletter
     }
-  },
-  { token: '' }
-  )
+  }, {
+    token: ''
+  })
 }
 
-function login () {
+async function login () {
+  const schema = [{
+    type: 'input',
+    message: 'email:',
+    name: 'email'
+  },
+  {
+    type: 'password',
+    message: 'Password:',
+    name: 'password'
+  }
+  ]
+  const result = await inquirer.prompt(schema)
   return new Promise((resolve, reject) => {
-    const schema = {
-      properties: {
-        email: {
-          required: true
-        },
-        password: {
-          hidden: true
-        }
-      }
-    }
-
-    prompt.start()
-
-    prompt.get(schema, function (err, result) {
-      if (err) {
-        reject(err)
-      } else {
-        authenticate(result.email, result.password).then((token) => {
-          resolve(token)
-        }).catch(err => {
-          reject(err)
-        })
-      }
+    authenticate(result.email, result.password).then((token) => {
+      resolve(token)
+    }).catch(err => {
+      reject(err)
     })
   })
 }
 
-function wantsNewsletter () {
+async function wantsNewsletter () {
+  const schema = [{
+    type: 'input',
+    message: 'Subscribe to the newsletter ([y]es or [n]o) ?',
+    name: 'wantsNewsletter',
+    choices: ['y', 'n'],
+    default: 'y'
+  }]
+  const result = await inquirer.prompt(schema)
   return new Promise((resolve, reject) => {
```

Comment:
replace this return by (and remove the new Promise(...)):

`return result.wantsNewsletter === 'y' ? 1 : 0`